Nos/Service/Relay/Filter.swift

```diff
@@ -113,8 +150,7 @@ struct Filter: Hashable, Identifiable {
             pTags.joined(separator: ","),
             search ?? "nil",
             since?.timeIntervalSince1970.description ?? "nil",
-            until?.timeIntervalSince1970.description ?? "nil",
-            inNetwork.description,
+            until?.timeIntervalSince1970.description ?? "nil"
```

Comment:
I did it the other way around: the hash function uses just the id now.

Reply:
What led you to do it the other way around? It looks to me like it would have been simpler to just add a `hasher.combine(keepSubscriptionOpen)` line up there than to have to get its description down here. And the other more complex lines here could be removed, too, which would be nice. But I didn't try it and might be missing something.

Reply:
I saw it implemented that way in https://medium.com/@saradrada/hashable-vs-identifiable-protocols-in-swift-71bd310c50cb with the hash function dependent on the id property. But I'll update it to match your preference.